models/news_models.js

```diff
@@ -28,4 +28,15 @@ exports.fetchUsers = async () => {
 exports.updateArticleById = async ( { article_id }, { inc_votes } ) => {
     const result = await db.query('UPDATE articles SET votes = votes + $1 WHERE article_id=$2 RETURNING *;', [inc_votes, article_id])
     return result.rows
+}
+
+exports.fetchArticles = () => {
+    
+    return Promise.all( [ getAllArticles(), getCommentCountAllArticles() ] )
```

Comment:
this has been slightly over complicated, you can add a comment count with a refactor of your sql query.